src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1607,6 +1613,12 @@ public Category updateCategory(Category category) {
                 JSON, json(Collections.singletonMap("category", category))), handle(Category.class, "category")));
     }
 
+    public Translation createCategoryTranslation(Long categoryId, Translation translation) {
+        checkHasCategoryId(categoryId);
```

Comment:
Why do we validate the Id of the category while in updateCategoryTranslation we are verifying the Id of translation ?